src/build.zig

```diff
@@ -122,6 +122,13 @@ fn compileRaylib(b: *std.Build, target: std.Build.ResolvedTarget, optimize: std.
                     .X11 => {
                         raylib.defineCMacro("_GLFW_X11", null);
                         raylib.linkSystemLibrary("X11");
+                        raylib.linkSystemLibrary("Xcursor");
+                        raylib.linkSystemLibrary("Xrender");
+                        raylib.linkSystemLibrary("Xrandr");
+                        raylib.linkSystemLibrary("Xinerama");
+                        raylib.linkSystemLibrary("Xi");
+                        raylib.linkSystemLibrary("Xext");
```

Comment:
Is this library really requried?

Reply:
yes

Reply:
XInput2.h includes X11/extensions/Xge.h which is part of Xext